test/AddTacticDoc.lean

```diff
@@ -0,0 +1,50 @@
+import Mathlib.Tactic.AddTacticDoc
+open Lean Elab
+
+/--My docstring for :one-/
+def one   := 1
+def two   := 2
+/--My docstring for :three-/
+def three := 3
+
+def tde : TacticDocEntry := {
+  name       := "something outlandish",
+  category   := DocCategory.hole_cmd,
+  decl_names := [`one],
+  tags       := ["not", "necessarily", "existent"]
+}
+
+def tde2 : TacticDocEntry := {
+  name       := "something outlandish",
+  category   := DocCategory.hole_cmd,
+  decl_names := [`one],
+  inherit_description_from := `three
+}
+
+-- 1. Test add_tactic_doc: use meta docstring
+/--meta!-/
+add_tactic_doc tde
+
+#eval show TermElabM _ from do
+  let tdes ← tactic.getTacticDocEntries
+  guard (tdes.length == 1)
+  if let some tde := tdes[0]? then
+    guard (tde.snd == "meta!")
+
+-- 2. Test add_tactic_dic: use the docstring of a single declaration from decl_names
```

Comment:
Should this be `add_tactic_doc` instead of `add_tactic_doc`?

Reply:
😔 yes